ofl/chivo/METADATA.pb

```diff
@@ -3,78 +3,34 @@ designer: "Omnibus-Type"
 license: "OFL"
 category: "SANS_SERIF"
 date_added: "2011-12-07"
-fonts {
-  name: "Chivo"
-  style: "normal"
-  weight: 300
-  filename: "Chivo-Light.ttf"
-  post_script_name: "Chivo-Light"
-  full_name: "Chivo Light"
-  copyright: "Copyright 2016 The Chivo Project Authors (omnibus.type@gmail.com)"
-}
-fonts {
-  name: "Chivo"
-  style: "italic"
-  weight: 300
-  filename: "Chivo-LightItalic.ttf"
-  post_script_name: "Chivo-LightItalic"
-  full_name: "Chivo Light Italic"
-  copyright: "Copyright 2016 The Chivo Project Authors (omnibus.type@gmail.com)"
-}
 fonts {
   name: "Chivo"
   style: "normal"
   weight: 400
-  filename: "Chivo-Regular.ttf"
-  post_script_name: "Chivo-Regular"
-  full_name: "Chivo Regular"
-  copyright: "Copyright 2016 The Chivo Project Authors (omnibus.type@gmail.com)"
+  filename: "Chivo[wght].ttf"
+  post_script_name: "Chivo-Thin"
+  full_name: "Chivo Thin"
```

Comment:
Could be nice if possible to preserve one of the previous postscript names (Light or Regular), if not, the Medium master could be taken as default instance cause closer to a regular weight